src/amo/reducers/userAbuseReports.js

```diff
@@ -0,0 +1,175 @@
+/* @flow */
+import type { UserType } from 'amo/reducers/users';
+
+export const HIDE_USER_ABUSE_REPORT_UI = 'HIDE_USER_ABUSE_REPORT_UI';
+export const LOAD_USER_ABUSE_REPORT = 'LOAD_USER_ABUSE_REPORT';
+export const SEND_USER_ABUSE_REPORT = 'SEND_USER_ABUSE_REPORT';
+export const SHOW_USER_ABUSE_REPORT_UI = 'SHOW_USER_ABUSE_REPORT_UI';
+
+
+type HideUserAbuseReportUIType = {|
+  user: UserType,
+|};
+
+export function hideUserAbuseReportUI(
+  { user }: HideUserAbuseReportUIType = {}
+) {
+  if (!user) {
+    throw new Error('user is required');
+  }
+
+  return {
+    type: HIDE_USER_ABUSE_REPORT_UI,
+    payload: { user },
+  };
+}
+
+type LoadUserAbuseReportType = {|
+  message: string,
+  reporter: Object | null,
+  user: {|
```

Comment:
Should this be `UserType`?

Reply:
No, this is just a subset of data returned by the API for the reported user: https://addons-server.readthedocs.io/en/latest/topics/api/abuse.html#submitting-a-user-abuse-report